content/docs/fairing/gcp-ai-platform-notebook.md

```diff
@@ -0,0 +1,121 @@
++++
+title = "Train and Deploy on GCP from an AI Platform Notebook"
+description = "Use Kubeflow Fairing to train and deploy a model on Google Cloud Platform (GCP) from a notebook that is hosted on Google Cloud AI Platform"
+weight = 35
++++
+
+This guide introduces you to using [Kubeflow Fairing][fairing-repo] to train and
+deploy a model to Kubeflow on [Google Kubernetes Engine (GKE)][gke] and [Google AI 
```

Comment:
We need to standardize on a long and short name for Cloud AI Platform. This page uses "Google AI Platform", "Google Cloud AI Platform", and "AI Platform".

Reply:
Thank you for pointing that out. I have updated the doc to use Google AI Platform as the long name and AI Platform as the short name.